### PR TITLE
ISSUE-135 Tmail backend docker images

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

**Comment by Arsnael** [comment]
=> https://hub.docker.com/r/linagora/tmail-backend

File: Jenkinsfile [hunk 1]
```diff
@@ -40,21 +40,21 @@ pipeline {
           }
           steps {
             script {
-              env.DOCKER_TAG = 'openpaas-branch-master'
+              env.DOCKER_TAG = 'branch-master'
```
Comment (Arsnael) [inline Jenkinsfile:43]: Opened to suggestions here :
* branch-master ?
* latest ?
* ...?

**Comment by Arsnael** [comment]
The first push of docker images to this new repo has been successful btw :)

File: Jenkinsfile (hunk 1 above)
Comment (chibenwa) [reply Jenkinsfile:43]: branch-master :+1:

**Review by chibenwa — COMMENTED** [review]
Looks good to me.

You have the rights to create the repos?

**Comment by Arsnael** [comment]
> You have the rights to create the repos?

Told you I got the rights given, I created the repo already, as I commented above: https://hub.docker.com/r/linagora/tmail-backend

**Comment by Arsnael** [comment]
Just forced-pushed to remove the DO NOT MERGE commit, that was just there to test the change on docker image pushes

File: Jenkinsfile [hunk 2]
```diff
@@ -40,21 +40,21 @@ pipeline {
...
                 env.DOCKER_TAG = env.TAG_NAME
               }
 
               echo "Docker tag: ${env.DOCKER_TAG}"
 
               // Temporary retag image names
-              sh "docker tag linagora/openpaas-james-memory linagora/james-memory:${env.DOCKER_TAG}"
-              sh "docker tag linagora/openpaas-james-distributed linagora/james-rabbitmq-project:${env.DOCKER_TAG}"
-              sh "docker tag linagora/openpaas-james-distributed-es6-backport linagora/james-rabbitmq-project:${env.DOCKER_TAG}-esv6"
+              sh "docker tag linagora/openpaas-james-memory linagora/tmail-backend:memory-${env.DOCKER_TAG}"
```
Comment (chibenwa) [inline Jenkinsfile:51]: Can we modify directly the JIB set up to get the right images there too?

Deployment tests will likely be affected too.

Comment (Arsnael) [reply Jenkinsfile:51]: I could look at it but I am not sure... as all our images on our repo have the same name with a different tag, I'm not sure how messy that would be on JIB side.

Comment (Arsnael) [reply Jenkinsfile:51]: Or do you mean the name change in the jib like :
`linagora/openpaas-james-memory` -> `linagora/tmail-backend-memory`

Comment (chibenwa) [reply Jenkinsfile:51]: Yes

**Review by chibenwa — APPROVED** [review]

**Review by vttranlina — APPROVED** [review]

**Pull request merged**